apps/website/src/theme/Toggle/index.tsx

```diff
@@ -49,7 +49,7 @@ function GithubLink() {
 
 export default function () {
   return (
-    <div className={styles.container}>
+    <div className={styles['container']}>
```

Comment:
@layershifter  is there a reason the tsconfig you originally proposed bans `styles.container` ?

Reply:
Not really, AFAIR it's defaults from Nx setup